haskell/src/Lentille/GraphQL.hs

```diff
@@ -110,7 +110,7 @@ streamFetch client mkArgs transformResponse = go Nothing
         lift $
           fetchWithLog
             (doGraphRequest client)
-            (mkArgs . fromMaybe (error "Missing endCursor") $ (Just . endCursor) =<< pageInfoM)
+            (mkArgs $ endCursor =<< pageInfoM)
```

Comment:
We might want to still fail if the endCursor is missing from the pageInfo, e.g.:

```suggestion
            (mkArgs $ (Just . fromMaybe (error "Missing endCursor from page info") . endCursor) =<< pageInfoM)
```